### PR TITLE
Act Model

### LINKED ISSUE #1: Model ActDTo and ContentItemDTo
Two separate Pull Requests, one for each!

Create the models for Act and ContentItem from the Valorant API 

https://developer.riotgames.com/apis#val-content-v1

### PR DESCRIPTION
**Pull request by JC00K**
- Model Created
- Separate Branch Pushed

### CODE REVIEW

File: src/models/Act.ts
Comment (stevenmherring) [inline src/models/Act.ts]: Adding comments like so usually doesn't require it's own commit, especially when it's follows immediately after the commit adding functionality.

`git commit --amend` allows you to add to the previous commit. Do everything else the same, make changes, git add, and replace your commit with the amend command.
https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History

File: src/models/Act.ts
Comment (stevenmherring) [inline src/models/Act.ts]: Use `export interface Act` or `export Class Act` I'll leave discovery and differentiation of the two to you to identify.
Here's a quick article to get started, https://ultimatecourses.com/blog/classes-vs-interfaces-in-typescript

File: src/models/Act.ts
Comment (stevenmherring) [inline src/models/Act.ts]: types almost always should be using `string`, `boolean`, the lower case primitives are preferred. Again, I'll let you look into this :)
https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string

**Review by stevenmherring — CHANGES_REQUESTED** [review]

**Comment by stevenmherring** [comment]
Need to rebase main here, also use what you learned from #4 to update this model :)

File: src/models/Act.ts
Comment (JC00K) [reply src/models/Act.ts]: Apologies. I'd had actually already updated this but did not push it with the correct changes.

File: src/models/Act.ts
Comment (JC00K) [reply src/models/Act.ts]: Apologies. I'd had actually already updated this but did not push it with the correct changes.

**Comment by stevenmherring** [comment]
rebuild dependencies
`npm install`
`git add package-lock.json`
`git commit --amend`
`git push -u origin Act -f`

**Comment by JC00K** [comment]
For some reason, this is giving me issues. There shouldn't be a conflict because there's no package-lock in the most recent push. I'm not able to resolve the conflict anymore as well.

Edit - Resolve conflicts now accessible, may have to through every conflict manually

Further Edit - Have manually gone through the package-lock.json and removed duplicates

**Comment by stevenmherring** [comment]
lets sync up on voice and walk through it. I would prefer we don't merge main into feature branches, because when that feature branch makes it back into main we will have a commit `merge main into Act`, main shouldn't have history of those branches.

**Review by stevenmherring — APPROVED** [review]

**Pull request merged**
